src/data/ores/stone.js

```diff
@@ -8,7 +8,7 @@ const { freeze } = Object
  */
 export const stone = freeze({
   description: 'A piece of rock.',
-  doesPriceFluctuate: true,
+  doesPriceFluctuate: false,
```

Comment:
when reviewing this i started thinking the stone shouldn't fluctuate in price, but the other ores still should. open to changing this back though

Reply:
This seems reasonable to me!